components/UserAccount/UserDetails.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from 'styled-components'
+import { FormButton } from '../FormElements/formElementsStyle'
+import { useUser } from '../UserAuth/User'
+
+export default function UserDetails() {
+  const user = useUser()
+  return (
+    <>
+      <UserCard>
+        <UserName>{user.name}</UserName>
+        <Achievements>
+          <UserParag>
+            You attended {user.bookings.length} classes since you joined
+          </UserParag>
+          <Membership>current membership info</Membership>
+        </Achievements>
+        <FormButton>delete account</FormButton>
+      </UserCard>
+    </>
+  )
+}
+
+const UserCard = styled.article`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 70%;
+  height: 100%;
+  margin: 1rem auto;
+  background-color: ${({ theme }) => theme.bg.secondary};
+  box-shadow: 3px 3px 2px 1px rgba(0, 0, 255, 0.2);
+  padding: 1rem;
+`
+
+const UserName = styled.h2`
+  border-radius: 50%;
+  width: 150px;
+  height: 150px;
+  background-color: ${({ theme }) => theme.bg.primary};
+  color: ${({ theme }) => theme.text.primary};
+  box-shadow: 3px 3px 2px 1px rgba(0, 0, 255, 0.2);
+  display: flex;
```

Comment:
I'm wondering my `h2` needs a `display: flex`. 
Can't you just use `text-align: center`?

Reply:
its not enough to align the text in the middle of the circle

Reply:
In that case: Leave the styling for the userName intact. Only style here text related (like font-size, color, etc) and wrap the `UserName` around a component which is the circle
e.g.

```javascript
<Circle>
  <Username>
</Circle>
```